herepy/routing_api.py

```diff
@@ -448,7 +448,7 @@ def route_v8(
         via_keys = []
         if via:
             for i, v in enumerate(via):
-                key = str.format("{0}{1}", "via", i)
+                key = str.format("{0}{1}_", "via", i)
```

Comment:
What is the purpose of using `_` here?

Reply:
If you have 11 via points, you will have `via1` and `via10` in `data` and `manipulation_keys`

The first url built in `__get` will contain: `via1=...&via10=..`

Then we loop to replace those `manipulation_keys` by the one needed by here (`via`):

- `url.replace("via1", "via")` -> `via=...&via0=...` (it also replaces the first part of `via10`)
- `url.replace("via10", "via") `-> `via=...&via0=...` (it does nothing)

By creating `via1_` and `via10_` keys instead:

given the url containing `via1_=...&via10_=..`

- `url.replace("via1_", "via")` -> `via=...&via10_=...` (we replace only the first key)
- `url.replace("via10", "via")` -> `via=...&via=...` (we replace the second key)

Reply:
Thanks for the clarification 👍 changes look good to me.

Reply:
Would you mind bumping the version of `codecov` to make the CI happy?

Reply:
Thanks for bumping the codecov version, it seems like your test is failing. Let's fix it and then we can merge.

Reply:
Responses requires Python 3.8 or higher as I see, can you remove the previous versions from CI config?

Reply:
do you want me to remove it from `ci.yml` and `travis.yml`, but keep python_requires=">=3.5" in setup?

Reply:
Yes please, 3.5 has already reached end of life. It's a good time to stop supporting it. Please update the setup.py too.